server/tests/backends/fixtures/aggregate/first_pypika.json

```diff
@@ -0,0 +1,88 @@
+{
+  "exclude": [
+    "mongo",
+    "mysql",
+    "pandas",
+    "postgres",
+    "snowflake"
+  ],
+  "step": {
+    "pipeline": [
+      {
+        "name": "aggregate",
+        "on": [
+          "beer_kind"
+        ],
+        "aggregations": [
+          {
+            "newcolumns": [
+              "first_name"
+            ],
+            "aggfunction": "first",
+            "columns": [
+              "name"
+            ]
+          }
+        ],
+        "keepOriginalGranularity": false
+      }
+    ]
+  },
+  "expected": {
+    "schema": {
+      "fields": [
+        {
+          "name": "beer_kind",
+          "type": "any"
+        },
+        {
+          "name": "first_name",
+          "type": "string"
+        }
+      ],
+      "pandas_version": "1.4.0"
+    },
+    "data": [
+      {
+        "beer_kind": "Belge blonde forte & Golden Ale",
+        "first_name": "Pauwel Kwak"
+      },
+      {
+        "beer_kind": "Best-sellers",
+        "first_name": "Ardwen Blonde"
+      },
+      {
+        "beer_kind": "Blanche & Weizen",
+        "first_name": "Weihenstephan Hefe Weizen Alcoholarm"
+      },
+      {
+        "beer_kind": "Blonde",
+        "first_name": "Brasserie De Sutter Brin de Folie"
+      },
+      {
+        "beer_kind": "Blonde",
+        "first_name": "Brasserie De Sutter Brin de Folie"
+      },
+      {
+        "beer_kind": "Blonde",
+        "first_name": "Brasserie De Sutter Brin de Folie"
+      },
```

Comment:
Shouldn't we only have one "Blonde" here since we're grouping on `beer_kind` and `keepOriginalGranularity` is false ?